database/migrations/2024_09_13_143830_realops_flight_add_callsign.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void {
+        Schema::table('realops_flights', function ($table) {
+            $table->string('callsign')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void {
+        Schema::table('realops_flights', function ($table) {
+            $table->dropColumn('string');
```

Comment:
This should drop 'callsign' not 'string'

Reply:
Fixed - thank you!